tests/tests_default.yml

```
@@ -4,10 +4,10 @@
   hosts: all
   gather_facts: false  # test that role works in this case
```

```suggestion
  gather_facts: false  # test that role works in this case
  vars:
    snapshot_lvm_action: check
```
since `snapshot_lvm_action` is a required parameter